files/es/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md

```diff
@@ -3,7 +3,6 @@ title: >-
   ¿Cuál es la diferencia entre la página web, el sitio web, el servidor web y el
   motor de búsqueda?
```

Comment:
This format may need a clean-up and update `_redirect`.

Reply:
`_redirect` is only updated by bot or scripts, not manually, the same for `original_slug` that it is put it in by the bot and it is for traking purposes